packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+import bgImage from './background.png';
+import DesktopIcon from '../images/DesktopIcon.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
+import { WelcomeUtils } from './welcome-utils';
+
+let showWelcome;
+
+$: showWelcome = false;
+
+onMount(async () => {
+  const welcomeUtils = new WelcomeUtils();
+  const ver = await welcomeUtils.getVersion();
+  if (ver === undefined) {
+    welcomeUtils.updateVersion('initial');
+    showWelcome = true;
+  }
+});
+</script>
+
+{#if showWelcome}
+  <div
+    class="flex flex-col flex-auto fixed top-10 left-0 right-0 bottom-0 bg-zinc-700 overflow-y-auto z-50"
+    style="background-image: url({bgImage}); background-position: 50% -175%; background-repeat: no-repeat; background-size: 100% 75%">
+    <!-- Header -->
+    <div class="flex flex-row flex-none backdrop-blur p-6">
+      <div class="flex flex-auto text-lg font-bold">Get started with Podman Desktop</div>
+    </div>
+
+    <!-- Body -->
+    <div class="flex flex-col justify-center content-center flex-auto backdrop-blur p-2">
+      <div class="flex justify-center p-2"><DesktopIcon /></div>
+      <div class="flex justify-center text-lg font-bold p-4">🎉 &nbsp;Welcome to Podman Desktop!</div>
```

Comment:
```suggestion
      <div class="flex justify-center text-lg font-bold p-4"><span class="mr-2">🎉</span>Welcome to Podman Desktop!</div>
```
just a suggestion, not mandatory maybe want to use tailwind there ?

Reply:
Fixed in second commit.